app/README.md

```diff
@@ -0,0 +1,27 @@
+# Proj
```

Comment:
Does this conflict with the original README? They are very similar.

Reply:
It shouldn't because they are in different directories